unicorn_mode/build_unicorn_support.py

```diff
@@ -59,15 +67,31 @@ def detect_from_env_or_file(target: str):
     print("[!] Please compile AFL++ first.")
     exit(1)
 
+upgraded = False
 if not shutil.which("cargo"):
     print("[!] No cargo, please install Rust in advance.")
     print("[!] TLDR: `curl https://sh.rustup.rs -sSf | sh -s -- -y`")
-    exit(1)
-
+    exit(2)
+else:
+    if not detect_rustc_version():
+        print("[!] Your rustc seems too old to build unicornafl and libafl")
+        
+        if shutil.which("rustup"):
+            print("[*] We will help you upgrade your rust version")
+            run_cmd("rustup upgrade", None, False)
+            run_cmd("rustup default stable", None, False)
```

Comment:
We shouldn't do this, just notify the user about what they have to do. 
Some users may manually install rust from apt, too

Reply:
Okay, make sense. Will put an error in later building process.